### PR TITLE
Support for readahead count when creating new volumes

### PR DESCRIPTION
**Pull request by jayc89**

### CODE REVIEW

**Comment by wallies** [comment]
+1 for getting this merged, as I dont really want to be changing the repo and need this setting for mongo servers

**Comment by mhaskel** [comment]
@jayc89 thanks for the contribution! In general this looks good, but I have one question which I'll put in an inline comment. If you can respond/fix that as relevant we should be good to merge this.

File: lib/puppet/type/logical_volume.rb [hunk 1]
```diff
@@ -77,6 +77,15 @@
     end
   end
 
+  newparam(:readahead) do
+    desc "The readahead count to use for the new logical volume."
+    validate do |value|
+      unless value =~ /^([0-9]+|Auto|None)/i
```
Comment (mhaskel) [inline lib/puppet/type/logical_volume.rb]: Does lvcreate care about the casing for auto or none? If it does this parameter should have a munge block to fix the case.

Comment (seif) [inline lib/puppet/type/logical_volume.rb]: I just tried lvcreate with different casing of nOnE and AuTo, it doesn't care.

**Comment by seif** [comment]
Would be great if this gets merged indeed! I just needed to set readahead and found this pull request!

**Comment by mhaskel** [comment]
Great, thanks for the contribution @jayc89 !

**Comment by seif** [comment]
@mhaskel sorry to pester, any ETA on when this would get pushed to the forge?

**Comment by JamieCressey** [comment]
Hey @mhaskel, any idea when the next release will be cut? Thanks!

**Comment by mhaskel** [comment]
Woops, sorry @seif @JamieCressey, I missed these pings. I'm not sure if we can get this out next week, but we should be able to do the following week at the latest. Thanks!

**Comment by seif** [comment]
@mhaskel no worries, using git ref for now.

**Pull request merged**